C++ API parity: CELU

pbelevich opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27309 C++ API parity: MultiheadAttention
* #27382 C++ API parity: Linear
* #27538 C++ API parity: Threshold
* #27537 C++ API parity: Tanhshrink
* #27536 C++ API parity: Tanh
* #27535 C++ API parity: Softsign
* #27534 C++ API parity: Softshrink
* #27489 C++ API parity: Softplus
* #27488 C++ API parity: Sigmoid
* **#27487 C++ API parity: CELU**

Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

pbelevich committed:
C++ API parity: CELU

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"




[ghstack-poisoned]

yf225 reviewed (approved):
Thanks for the awesome work @pbelevich!

pbelevich committed:
Update on "C++ API parity: CELU"


Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"


Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"


Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"


Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"


Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"


Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"


Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"


Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: CELU"


Differential Revision: [D17835406](https://our.internmc.facebook.com/intern/diff/D17835406)

[ghstack-poisoned]
